webui/react/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "webui-react",
   "version": "0.1.0",
   "private": true,
+  "homepage": "/det",
```

Comment:
**suggestion:** should this just be `.` to serve the assets and relative paths the same as where `index.html` is served? Maybe it's not as simple as that...
context: https://stackoverflow.com/a/44048328/5402432

Reply:
I removed it we don't need it